tutorials/data-lake-file-containers-hdlfscli/data-lake-file-containers-hdlfscli.md

```diff
@@ -102,7 +102,9 @@ Some HDLFSCLI help documentation should appear if it is successfully installed a
 [ACCORDION-END]
 
 [ACCORDION-BEGIN [Step 3: ](Generate Certificates)]
-To connect the HDLFSCLI to a HANA Data Lake file container, a certificate will need to be generated to make a secure connection. Below are the steps required to create a self-signed certificate to get started using the HDLFSCLI. You will require an installation of OpenSSL. Use your preferred Linux package installer to install OpenSSL if it is not already installed. If you're using a Windows machine, then Windows Subsystem Linux will have OpenSSL installed. Alternatively, OpenSSL can be installed for Windows. Then, follow these steps to creating your self-signed certificate.
+To connect the HDLFSCLI to a HANA Data Lake file container, a certificate will need to be generated to make a secure connection. Below are the steps required to create a self-signed certificate to get started using the HDLFSCLI. You will require an installation of OpenSSL. Use your preferred Linux package installer to install OpenSSL if it is not already installed. If you're using a Windows machine, then Windows Subsystem Linux will have OpenSSL installed. Alternatively, OpenSSL can be installed for Windows. OpenSSL for Windows can be downloaded from [Here](https://slproweb.com/products/Win32OpenSSL.html).  
+
+Then, follow these steps to creating your self-signed certificate.
 
 **Note**: Make sure the certificate fields are not all exactly the same between the Certificate Authority (CA) and client certificates, otherwise it is assumed to be a self-signed cert and the cert validation below will fail.
```

Comment:
Please adjust formatting as shown in the style guide. Don't start a note with the label "Note". Please do this for all notes with wrong formatting.
https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9#mcetoc_1dtqfqchq21